src/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -453,13 +453,27 @@ public static void X509CertificateCollectionAsIList()
                 il.Add(c2);
 
                 Assert.Throws<ArgumentNullException>(() => il[0] = null);
+            }
+        }
+
+       [Fact]
+       // On Desktop, list is untyped, and X509Certificate2 is not disposable
```

Comment:
I'm not sure this test ever passed on desktop. In fact none of these ever passed because it's not disposable on desktop and they all have using statements.